go/test/endtoend/cluster/vtctld_process.go

```diff
@@ -64,9 +64,6 @@ func (vtctld *VtctldProcess) Setup(cell string, extraArgs ...string) (err error)
 		"--service_map", vtctld.ServiceMap,
 		"--backup_storage_implementation", vtctld.BackupStorageImplementation,
 		"--file_backup_storage_root", vtctld.FileBackupStorageRoot,
-		// hard-code these two soon-to-be deprecated drain values.
-		"--wait_for_drain_sleep_rdonly", "1s",
-		"--wait_for_drain_sleep_replica", "1s",
```

Comment:
are these flags now deleted? or only deprecated?

Reply:
They're now deleted. They were marked as deprecated before but this end2end test comment wasn't updated: inhttps://github.com/vitessio/vitess/blob/23727bfee12607250da769180fab8064b073dad1/go/vt/wrangler/keyspace.go#L48-L52